ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/computation/StatementExecutorTest.java

```diff
@@ -626,9 +692,10 @@ private void terminateQueries() {
         new CommandId(CommandId.Type.TABLE, "_TerminateGen", CommandId.Action.CREATE);
     statementExecutor.handleStatement(
         terminateCommand2, terminateCommandId2, Optional.empty());
-    final Optional<CommandStatus> terminateCommandStatus2 =
-        statementExecutor.getStatus(terminateCommandId2);
-    assertThat(terminateCommandStatus2.get().getStatus(), equalTo(CommandStatus.Status.SUCCESS));
+    assertThat(getCommandStatus(terminateCommandId2), equalTo(CommandStatus.Status.SUCCESS));
   }
 
+  private CommandStatus.Status getCommandStatus(CommandId commandId) {
+    return statementExecutor.getStatus(commandId).get().getStatus();
```

Comment:
```suggestion
    final Optional<CommandStatus> status = statementExecutor.getStatus(commandId);
    assertThat("command not registered:" + commandId, status, is(not(Optional.empty()));
    return status.get().getStatus();
```

Just gives a better error message when things don't work.